### PR TITLE
MINOR: Bump system test version from 2.2.1 to 2.2.2

### PR DESCRIPTION
**Pull request by rhauch**
*More detailed description of your change,
if necessary. The PR title and PR message become
the squashed commit message, so use a separate
comment to ping reviewers.*

*Summary of testing strategy (including rationale)
for the feature or bug fix. Unit and/or integration
tests are expected for any behaviour change and
system tests should be considered for larger changes.*

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

**Comment by rhauch** [comment]
System test run with this PR: https://jenkins.confluent.io/job/system-test-kafka-branch-builder/3597/

File: vagrant/base.sh
Comment (ijuma) [inline vagrant/base.sh]: Should we do 2.3.0 -> 2.3.1 at the same time?

Comment (rhauch) [reply vagrant/base.sh]: Ack.

**Review by ijuma — APPROVED** [review]
LGTM if the tests pass.

**Comment by rhauch** [comment]
Both JDK 11 builds passed, but the JDK 8 build passed with an error that IIUC does not attempt to use older Kafka versions:
```
kafka.server.ApiVersionsRequestTest > testApiVersionsRequestValidationV3 STARTED
kafka.integration.MetricsDuringTopicCreationDeletionTest.testMetricsDuringTopicCreateDelete failed, log available in /home/jenkins/jenkins-slave/workspace/kafka-pr-jdk8-scala2.11/core/build/reports/testOutput/kafka.integration.MetricsDuringTopicCreationDeletionTest.testMetricsDuringTopicCreateDelete.test.stdout

kafka.integration.MetricsDuringTopicCreationDeletionTest > testMetricsDuringTopicCreateDelete FAILED
    java.lang.AssertionError: assertion failed: UnderReplicatedPartitionCount not 0: 1
        at scala.Predef$.assert(Predef.scala:170)
        at kafka.integration.MetricsDuringTopicCreationDeletionTest.testMetricsDuringTopicCreateDelete(MetricsDuringTopicCreationDeletionTest.scala:123)
```

**Comment by rhauch** [comment]
Previous system test run failed with 2.2.2 TAR file not available on S3. Added the missing artifacts to the S3 bucket, and running again: https://jenkins.confluent.io/job/system-test-kafka-branch-builder/3609/

**Comment by rhauch** [comment]
Build 3609 failed on one test:
```
Module: kafkatest.tests.core.round_trip_fault_test
Class:  RoundTripFaultTest
Method: test_produce_consume_with_client_partition
```
Rerunning as https://jenkins.confluent.io/job/system-test-kafka-branch-builder/3615/

**Comment by rhauch** [comment]
Last system test failure resulted in 1 failure:
```
Module: kafkatest.tests.connect.connect_distributed_test Class:  ConnectDistributedTest Method: test_pause_state_persistent Arguments: {   "connect_protocol": "sessioned" }
```
This appears to be a flaky test and timeout related (see #7790, #7789 for fixes), and not one that uses older broker versions. So this PR shouldn't affect that test.

I'll wait to merge until those two PRs are merged, however.

**Pull request merged**
